test/sequential/maintenance-pallet.api.rolldown.test.ts

```diff
@@ -262,15 +265,9 @@ describe.each(["mm", "upgradabilityMm"])(
         "%s operations are allowed in mm",
         async (testName) => {
           testLog.getLog().info("DEBUG::TestName - " + testName);
-          let signer: User;
           const [extrinsic, testsSigner] = tests[testName];
-          if (mmMode === "mm") {
-            signer = testsSigner;
-          } else {
-            signer = user;
-          }
-          const nonce = await getCurrentNonce(signer.keyRingPair.address);
-          await signTx(api, extrinsic, signer.keyRingPair, {
+          const nonce = await getCurrentNonce(testsSigner.keyRingPair.address);
+          await signTx(api, extrinsic, testsSigner.keyRingPair, {
```

Comment:
why this?

Reply:
In this test we try to leave sequencer, but **user** is not a sequencer here(as I suppose), so that's why I changed this one.

Reply:
it must be. , 
```
        await Sudo.batchAsSudoFinalized(
          await SequencerStaking.provideSequencerStaking(
            users[2].keyRingPair.address,
          ),
        ).then((value) => {
          testLog
            .getLog()
            .info("Sequencer staking provided" + JSON.stringify(value));
          expectMGAExtrinsicSuDidSuccess(value);
        });
```
but perhaps is failing, try removing all the sequencers beforeAll, I found some issues by tests hitting the limit of sequencers